src/components/breadcrumb/_breadcrumb.scss

```diff
@@ -31,28 +31,34 @@ $breadcrumb-chevron-height: 0.65rem;
         content: none;
       }
 
-      &:not(:nth-last-child(2)) {
+      &:not(:nth-last-child(1)) {
         display: inline-block;
       }
 
+      &:first-child:nth-last-child(1) {
+        &:before {
+          @include icon('chevron-left', $breadcrumb-chevron-height, $breadcrumb-chevron-height);
+          content: '';
+          margin: 0 0.2rem 0 0;
+          vertical-align: middle;
+        }
+      }
+
       &:not(:last-child):after {
         @include icon('chevron-right', $breadcrumb-chevron-height, $breadcrumb-chevron-height);
         content: '';
-        vertical-align: middle;
         margin: 0;
+        width: 1.25rem;
 
         // We have to override the icon settings so it renders correctly in ie11
         background-position: center center;
-        width: 1.25rem;
+        vertical-align: middle;
       }
     }
-
-    &--current {
-      color: $color-text-light;
-    }
   }
 
   &__link {
-    text-decoration: none;
+    color: #222;
```

Comment:
Use variable: 
```suggestion
    color: $color-text;
```

Reply:
...or probably a new variable for breadcrumbs?

Reply:
I think the breadcrumb is always going to be the text colour, so it makes sense to use the existing text variable.